README.md

```diff
@@ -366,6 +366,24 @@ var pin5 = M5StickC.GpioController.OpenPin(36, PinMode.Output);
 
 The M5Core, M5Core2, Fire and Atom Lite/Matrix exposes 2 DAC and you can access them thru the `Dac1` and `Dac2` properties. Refer to the [DAC documentation](https://github.com/nanoframework/System.Device.Dac) for more information.
 
+### I2S (sound)
+On the M5Core2, wav files can be played using I2S by installing the package [nanoFramework.System.Device.I2s](https://github.com/nanoframework/System.Device.I2S). 
+To play wav files on the M5Core2, use following initialization:
+```csharp
```

Comment:
In Markdown, the code blocks need to be separated by 1 extra line from the text or anything before and after.
```suggestion

```csharp
```